CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt

```diff
@@ -149,10 +151,13 @@ class GalleryFragment internal constructor() : Fragment() {
                             if (mediaList.isEmpty()) {
                                 fragmentManager?.popBackStack()
                             }
-                        }}
 
-                    .setNegativeButton(android.R.string.no, null)
-                    .create().showImmersive()
+                        }
+
+                        .setNegativeButton(android.R.string.no, null)
+                        .create().showImmersive()
+
+            } ?: Toast.makeText(activity, "No media file available", Toast.LENGTH_SHORT).show()
```

Comment:
I am not owner of this repository, but could you extract it to `strings.xml`?

Reply:
Sir I only resolved the issue that is mentioned in issue title. Secondly, yes we can do extract.

Reply:
:+1: